src/quo/animated.cljs

```diff
@@ -178,20 +178,20 @@
   (.withOffset ^js redash (clj->js config)))
 
 (defn with-spring-transition
-  [val config]
-  (.withSpringTransition ^js redash val (clj->js config)))
+  [v config]
+  (.withSpringTransition ^js redash v (clj->js config)))
```

Comment:
For this issue I think it's ok to call it `v` but for the following uses I'd prefer to call it `value`  or `animated-value`, just because it's more clear IMO.

I understand the main task is to avoid shadowed vars, and is hard to understand the context of all of these vars to give a more clear name.

Reply:
I agree with you. The reason I used `v` in this ns is because `value` is already taken, i.e. there's a function named `value` and I didn't want to rename it in this PR.